pkg/apis/pipeline/v1/result_validation.go

```diff
@@ -51,3 +53,25 @@ func (tr TaskResult) Validate(ctx context.Context) (errs *apis.FieldError) {
 
 	return nil
 }
+
+
+func validateObjectResult(tr TaskResult) (errs *apis.FieldError) {
```

Comment:
could you please add a docstring for this function?

Reply:
np!